app/views/layouts/_admin.html.haml

```diff
@@ -1,14 +1,20 @@
 .container-fluid
+  - if @conference and not @conference.short_title.nil?
```

Comment:
Actually i thought of this only but as we are using the condition `if @conference and not @conference.short_title.nil?` for the admin_sidebar  so i used it here also &#128517;. Should i push the changes ?